scipy/stats/_continuous_distns.py

```diff
@@ -9972,11 +9972,71 @@ class vonmises_gen(rv_continuous):
     `vonmises_line` is the same distribution, defined on :math:`[-\pi, \pi]`
     on the real line. This is a regular (i.e. non-circular) distribution.
 
-    `vonmises` and `vonmises_line` take ``kappa`` as a shape parameter.
+    Note about distribution parameters: `vonmises` and `vonmises_line` take
+    ``kappa`` as a shape parameter and the ``scale`` parameter does not have
+    any effect.
 
-    %(after_notes)s
+    Examples
+    --------
+    Import the necessary modules.
 
-    %(example)s
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.stats import vonmises
+
+    Define distribution parameters: the circular mean $\mu$ has to be passed
+    via the ``loc`` (location) parameter and the concentration parameter
+    $\kappa$ as ``kappa``.
+
+    >>> loc = 0.5 * np.pi
+    >>> kappa = 1
```

Comment:
Since $\mu$ isn't defined, the Greek letters aren't rendering, and this is explained just above in the notes, how about:
```suggestion
    Define distribution parameters.

    >>> loc = 0.5 * np.pi  # circular mean
    >>> kappa = 1  # concentration
```